pandora/models/pandora.py

```diff
@@ -223,11 +226,26 @@ class AdItem(PlaylistModel):
     def is_ad(self):
         return True
 
-    def register_ad(self, station_id):
-        self._api_client.register_ad(station_id, self.tracking_tokens)
+    def register_ad(self, station_id=None):
+        if not station_id:
+            station_id = self.station_id
+        if self.tracking_tokens:
+            self._api_client.register_ad(station_id, self.tracking_tokens)
+        else:
+            raise ParameterMissing('No ad tracking tokens provided for '
```

Comment:
I think raising the exception makes sense in anticipation of API consumers making their own ```register_ad``` calls for whatever reason, in which case they should be warned that they are not using it correctly.